Domain/ProductManagement/Inventory.cs

```diff
@@ -0,0 +1,261 @@
+﻿using LenaInventoryManagementSystem.Domain.ProductManagement;
+using System.ComponentModel;
+using System.ComponentModel.Design;
+
+namespace LenaInventoryManagementSystem.Domain
+{
+    public class Inventory
+    {
+        private static List<Product> products = new();
+        internal static void MainMenu()
+        {
+            Console.WriteLine(" -------------------------- ");
+            Console.WriteLine("| Select the action wanted |");
+            Console.WriteLine(" -------------------------- ");
+            Console.WriteLine("| 1: Add a new product     |");
+            Console.WriteLine("| 2: View all products     |");
+            Console.WriteLine("| 3: Update a product      |");
+            Console.WriteLine("| 4: Delete a product      |");
+            Console.WriteLine("| 5: View a product        |");
+            Console.WriteLine("| 6: Exit                  |");
+            Console.WriteLine(" -------------------------- ");
+
+            Console.WriteLine("Selected: ");
+            string? userSelection = Console.ReadLine();
+
+            switch (userSelection)
+            {
+                case "1":
+                    AddANewProduct();
+                    break;
+                case "2":
+                    ViewAllProducts();
+                    break;
+                case "3":
+                    UpdateAProduct();
+                    break;
+                case "4":
+                    DeleteAProduct();
+                    break;
+                case "5":
+                    //ViewAProduct();
+                    break;
+                default:
+                    Console.WriteLine("Invalid selection. Please try again.");
+                    break;
+
+            }
+        }
+
+        private static void AddANewProduct()
+        {
+
```

Comment:
NIT: extra new line, please remove :D please check this for the rest of files and remove extra lines ;)